jekyll/_cci2/orb-author.md

```diff
@@ -17,104 +17,125 @@ version:
 
 This orb authoring guide assumes you have read the [Introduction to orbs]({{site.baseurl}}/2.0/orb-intro) document, the [Introduction to authoring an orb]({{site.baseurl}}/2.0/orb-author-intro) document, and claimed your namespace. At this point you are ready to develop an orb.
 
-Whether you are writing your first orb or getting ready for production level, we recommend using our [orb development kit](#orb-development-kit) to get started. Alternatively, as orbs are packages of [reusable configuration]({{site.baseurl}}/2.0/reusing-config), they can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish), as singular `yaml` files, and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
+Whether you are writing your first orb or getting ready for production level, we recommend using our [Orb Development Kit](#orb-development-kit) to get started. Alternatively, as orbs are packages of [reusable configuration]({{site.baseurl}}/2.0/reusing-config), they can be written [manually]({{site.baseurl}}/2.0/orb-author-validate-publish), as singular `yaml` files, and published using our [circleci orb cli]({{site.baseurl}}/2.0/local-cli/#installation).
 
 ## Orb Development Kit
 {: #orb-development-kit }
 
-The orb development kit refers to a suite of tools that work together to simplify the orb development process, with automatic testing and deployment on CircleCI. The orb development kit is made up of the following components:
+The Orb Development Kit refers to a suite of tools that work together to simplify the orb development process, with automatic testing and deployment on CircleCI. The Orb Development Kit is made up of the following components:
 
-* [Orb Project Template](https://github.com/CircleCI-Public/Orb-Project-Template)
-* [Orb Pack]({{site.baseurl}}/2.0/orb-concepts/#orb-packing)
-* [Orb Init](https://circleci-public.github.io/circleci-cli/circleci_orb_init.html)
+* [Orb Template](https://github.com/CircleCI-Public/Orb-Template)
+* [CircleCI CLI](https://circleci-public.github.io/circleci-cli/)
+    * [Orb Pack Command]({{site.baseurl}}/2.0/orb-concepts/#orb-packing)
+    * [Orb Init Command](https://circleci-public.github.io/circleci-cli/circleci_orb_init.html)
 * [Orb Tools Orb](https://circleci.com/developer/orbs/orb/circleci/orb-tools)
 
 ### Getting started
```

Comment:
I feel like this page is a bit long. Maybe the getting started steps should be on a separate page to the rest of the info we have here.

Reply:
@KyleTryon I'm deleting this comment and I'm going to make a Jira ticket to look at splitting these docs into more sections, maybe: `Using Orbs` `Authoring Orbs` `Testing Orbs` `Publishing Orbs` or something.